docs/tools/vdb_table/data/oramasearch.json

```diff
@@ -108,24 +108,24 @@
     "comment": ""
   },
   "pricing": {
-    "value": "free plan / $4.99 per month (pro)",
+    "value": "free plan / $29.99 per month (pro)",
     "source_url": "",
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Orama can work completely in-memory in your browser or server"
   },
   "multi_tenancy": {
     "support": "full",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
```

Comment:
This is the meaning of this column: https://github.com/superlinked/VectorHub/blob/a986982150dfce541551a3a9a424b63b98920918/docs/tools/vdb_table/vendor.schema.json#L33

The index should be served from disk to qualify

Reply:
Yes, we support this by storing partitions on disk via the plugin I wrote down in the `source_url` field. Orama Cloud does that automatically for you, OSS orama requires you to perform this operation on your own